OmarEspinosaCastro/31-Agosto/Gradle/src/main/java/CurrencyFormatter.java

```diff
@@ -0,0 +1,27 @@
+import org.apache.logging.log4j.LogManager;
```

Comment:
The default unnamed package should not be used

Reply:
add the package